indico/modules/events/registration/forms.py

```diff
@@ -392,3 +393,14 @@ def __init__(self, event, **kwargs):
 
     def is_submitted(self):
         return super(BadgeSettingsForm, self).is_submitted() and 'submitted' in request.form
+
+
+class ImportRegistrationsForm(IndicoForm):
+    source_file = FileField(_("Source File"), accepted_file_types='.csv')
+    skip_moderation = BooleanField(_("Skip Moderation"), widget=SwitchWidget(), default=True,
+                                   description=_("If enabled, the registration will me immediately accepted"))
+    notify_users = BooleanField(_("E-mail users"), widget=SwitchWidget(),
+                                description=_("Whether the imported users should receive an e-mail notification"))
+    create_pending = BooleanField(_("Match missing users"), widget=SwitchWidget(), default=True,
```

Comment:
Is there any good reason to create pending users here? If a user registers later they are associated with the registration anyway, and in the user search we never show pending users. So I'd rather offer to create an EventPerson so they can find him when searching to add e.g. a speaker - but never a pending indico User

Reply:
And how do we phrase that option (EventPerson)?

Reply:
Then I'll just remove the option and set it to `False` by default. Anyway, the unification of Event Persons with registrations in version 2.1 should make this irrelevant.